modules/fscloud/version.tf

```diff
@@ -7,7 +7,7 @@ terraform {
     # tflint-ignore: terraform_unused_required_providers
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.56.1, < 2.0.0"
+      version = ">= 1.62.0"
```

Comment:
add back the higher range limit: `">= 1.62.0, <2.0.0"`

Reply:
Done